core/src/main/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpServiceFactory.java

```diff
@@ -162,19 +162,22 @@
 
     /**
      * Returns the list of {@link AnnotatedHttpService} defined by {@link Path} and HTTP method annotations
-     * from the specified {@code object}, {@link ExceptionHandlerFunction}'s,
-     * {@link RequestConverterFunction}'s and {@link ResponseConverterFunction}'s.
+     * from the specified {@code object}, {@link ExceptionHandlerFunction}s, {@link RequestConverterFunction}s,
+     * {@link ResponseConverterFunction}s and {@link AnnotatedHttpServiceExtensions}.
      */
     public static List<AnnotatedHttpServiceElement> find(
             String pathPrefix, Object object,
             List<ExceptionHandlerFunction> exceptionHandlerFunctions,
             List<RequestConverterFunction> requestConverterFunctions,
-            List<ResponseConverterFunction> responseConverterFunctions) {
+            List<ResponseConverterFunction> responseConverterFunctions,
+            AnnotatedHttpServiceExtensions extensions) {
```

Comment:
I think `AnnotatedHttpServiceExtensions` structure is similar to `exceptionHandlerFunctions` + `requestConverterFunctions` + `responseConverterFunctions`. Shouldn't we accept one of them? 

How about merging `extensions.XXXFunctions` into `XXXFunctions` before calling this method.

Reply:
I also worried about this for a while. If you think so, I will correct it. 😃

Reply:
+1